src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -77,7 +77,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String user = userService.getCurrentUser().getEmail();
 
-    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.basic());
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.basicWithImages());
```

Comment:
I think this will allow the user to type their own <img> tags, in addition to the automatically replaced ones below--is that what we want? What happens if you send a message like `<img src="https://upload.wikimedia.org/wikipedia/en/e/ed/Nyan_cat_250px_frame.PNG">`? Does it get double-replaced?

Reply:
this happens should it actually replace it with the image?
<img width="610" alt="Screen Shot 2019-03-12 at 6 42 19 PM" src="https://user-images.githubusercontent.com/37354543/54241251-bb81cb80-44f6-11e9-83b2-699efd8dfb56.png">

Reply:
Ah, maybe my fault for picking a link with all-caps PNG. Maybe try again with an image that works when you put just the URL by itself?

I think it's mostly personal preference for what behavior seems better for the product--right now on app engine, I think the basic whitelist will strip out the tag completely if you type it, but still auto-render image URLs.